src/nl/rubensten/texifyidea/lang/LatexNoMathCommand.kt

```diff
@@ -20,6 +20,7 @@ enum class LatexNoMathCommand(
     ADDTOCOUNTER("addtocounter", "countername".asRequired(), "value".asRequired()),
     A_RING("aa", display = "å"),
     CAPITAL_A_RING("AA", display = "Å"),
+    ADDBIBRESOURCE("addbibresource", RequiredFileArgument("bibliographyfile", "bib"), dependency = Package.BIBLATEX), // .bib extension is required
```

Comment:
Comment is redundant. Also add a static import for `Package.BIBLATEX`. Maybe also do this for other stray packages that have not been imported statically. Imo, adds to readability and reduces clutter in an already cluttered file.

Reply:
For hyperref and comment it's needed to distinguish between the command and the package of the same name. But I will clean up others.